internal/controller/frrconfiguration_controller.go

```diff
@@ -81,6 +77,19 @@ func (r *FRRConfigurationReconciler) Reconcile(ctx context.Context, req ctrl.Req
 	return ctrl.Result{}, nil
 }
 
+func (r *FRRConfigurationReconciler) applyEmptyConfig(req ctrl.Request) {
+	empty := frrk8sv1beta1.FRRConfiguration{}
+	config, err := apiToFRR(empty)
+	if err != nil {
+		level.Error(r.Logger).Log("controller", "FRRConfigurationReconciler", "failed to translate the empty config", req.NamespacedName.String(), "error", err)
+		return
+	}
+
+	if err := r.FRRHandler.ApplyConfig(config); err != nil {
+		level.Error(r.Logger).Log("controller", "FRRConfigurationReconciler", "failed to apply the empty config", req.NamespacedName.String(), "error", err)
+	}
```

Comment:
could this be transient or it's always something internal?

Reply:
given the current code, it's stateless
I am even tempted to panic here

Reply:
done